### PR TITLE
Update docs for payments app

### PR DESCRIPTION
**Pull request by shahbaz-mehar**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,37 +1,5 @@
...
-## [2.1.5]
-- Re-branding from Valitor to Altapay
-- API access token now supports 250 characters
-- Fix: Unable to decode characters for some locales
-## [2.1.0]
-- Added the possibility of customizing the select payment method and payment form pages
-- Multiple small fixtures
-## [2.0.0]
-- Multiple fixtures and improvements
 ## [1.0.0]
```
Comment (emicha) [inline CHANGELOG.md:4]: I would recommend not to delete this.
Instead, move it to a new section, called "HPSDK Payment App [deprecated]"

Comment (emicha) [reply CHANGELOG.md:4]: For the docs, move them to:
- docs/deprecated_HPSDK/...

Comment (shahbaz-mehar) [reply CHANGELOG.md]: @emicha I have updated the pull request based on the above suggestions.

**Review by emicha — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,11 @@
 # Changelog
 All notable changes to this project will be documented in this file.
 
+## [app-1.0.0]
+- First release of AltaPay Payments App
```
Comment (emicha) [inline CHANGELOG.md:5]: Please explain the features supported int he release notes, for example:

```suggestion
## [app-1.0.0]
- First release of using the new Payments App Framework https://shopify.dev/apps/payments
- Supported Features:
-- Payment Page Redirect
-- Auto-Expand Credit Card Form
...
```

Comment (shahbaz-mehar) [reply CHANGELOG.md]: @emicha I have added the Supported Features in the release notes.

**Review by emicha — APPROVED** [review]

**Pull request merged**